composer.json

```diff
@@ -14,7 +14,8 @@
     ],
 
     "require": {
-        "bolt/filesystem": "^2.0",
+        "bolt/filesystem": "^2.1",
+        "contao/imagine-svg": "^0.1.2",
```

Comment:
> nearly nobody actually deos that

You mean people don't break BC with `0.Y` releases or they do?

`^` has always had that logic. Are you saying we should use `~` here?